webapp/utils/text_formatting.jsx

```diff
@@ -162,12 +162,23 @@ export function autolinkAtMentions(text, tokens, usernameMap) {
         return (Constants.SPECIAL_MENTIONS.indexOf(u) !== -1 || Boolean(usernameMap[u]));
     }
 
+    function isSpecialMention(username) {
```

Comment:
If you change `mentionExists` to not check `Constants.SPECIAL_MENTIONS`, you won't need this function any more.